test/unix/main.ml

```diff
@@ -714,6 +714,8 @@ module Close = struct
     let* Context.{ rw; _ } =
       Context.with_full_index ~throttle:`Block_writes ~size:100 ()
     in
+    let root = Index.root rw in
```

Comment:
there is a `root` global variable already defined in this file, can you reuse it?

Reply:
Oh, yes, don't know how I missed it, thanks! And thanks for the review :-)